src/test/scala/outwatch/ScenarioTestSpec.scala

```diff
@@ -153,18 +177,16 @@ class ScenarioTestSpec extends UnitSpec with BeforeAndAfterEach {
       state = adds.merge(deletes)
         .scan(Vector[String]())((state, modify) => modify(state))
         .map(_.map(n => TodoComponent(n, deleteHandler)))
-
-
-      root <- div(
-        TextFieldComponent("Todo: ", inputHandler),
+      textFieldComponent <- TextFieldComponent("Todo: ", inputHandler)
+    } yield div(
```

Comment:
Correct me, if I'm wrong, but here root is a `IO[IO[VNode]]` correct?

Reply:
That is correct, it would end up flattened by the ``` vtree.flatMap``` below.